src/app/loginComponents/requests/organization-invite-confirm-dialog.html

```diff
@@ -0,0 +1,15 @@
+<h1 mat-dialog-title>{{data.approve ? 'Accept' : 'Decline'}} Organization</h1>
```

Comment:
If this is for accepting an invite, I think it should say you're accepting/declining the invitation, not the organization. 

Also, if you were invited to more than one org, which is probably unlikely, it would be nice to know the org name you are accepting.